File: Daily Challenge/C/Generate Passwords For Employees 2.9.2025.c
#include <stdio.h>

int main() {
    int N, C, D, copy;
    scanf("%d %d %d", &N, &C, &D);
    copy = C;

    for (int i = 1; i <= D; i++) {
        for (int j = 1; j <= N; j++) {
            printf("%d ", C + j);
        }
        printf("\n");
        C = copy + i;
    }
    return 0;
}

File: Daily Challenge/Java/Generate Passwords For Employees 2.9.2025.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int C = sc.nextInt();
        int D = sc.nextInt();
        int copy = C;

        for (int i = 1; i <= D; i++) {
            for (int j = 1; j <= N; j++) {
                System.out.print((C + j) + " ");
            }
            System.out.println();
            C = copy + i;
        }
    }
}